packages/model-typings/src/models/IModerationReportsModel.ts

```diff
@@ -27,7 +33,7 @@ export interface IModerationReportsModel extends IBaseModel<IModerationReport> {
 	countReportsInRange(latest: Date, oldest: Date, selector: string): Promise<number>;
```

Comment:
This should also be updated to `countMessageReportsInRange` (+ the code should be updated too to ensure we only count message reports)